mobile/src/screens/chat-list/chat-list.tsx

```diff
@@ -1,41 +1,41 @@
-import React, { type ReactNode, useCallback } from 'react';
+import React, { useCallback } from 'react';
 import LinearGradient from 'react-native-linear-gradient';
 
 import PlusIcon from '#assets/img/icons/plus.svg';
 import {
   Card,
+  Header,
   InputSearch,
   Link,
   ScrollView,
-  Text,
   View,
 } from '#libs/components/components';
 import { AppColor, RootScreenName } from '#libs/enums/enums';
-import { useEffect, useNavigation, useSearch } from '#libs/hooks/hooks';
+import { ChatScreenName } from '#libs/enums/navigation/chat-screen-name.enum';
+import { useEffect, useSearch } from '#libs/hooks/hooks';
+import { type NavigationScreenProperties } from '#libs/types/types';
 
-import { Badge } from './components/components';
 import mockedChats from './libs/data.json';
 import { styles } from './styles';
 
 const mockedCount = 12;
 
-const ChatList: React.FC = () => {
-  const navigation = useNavigation();
-
+const ChatList = ({
+  navigation,
+  route,
+}: NavigationScreenProperties): JSX.Element => {
```

Comment:
```suggestion
const ChatList: React.FC<NavigationScreenProperties> = ({
  navigation,
  route,
}) => {
```
We use this if we don't have generic type. It laso relates to `Chat` component

Reply:
If I do it this way, I get this error: 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/29295c7c-8353-4383-95e3-2dd734412d09)

Reply:
Oh, okay, leave it as it was